### DISCUSSION TITLE
Use car GPS for live tracking?

### DISCUSSION THREAD

**Discussion by Bastjen** [BimmerGestalt/AAIdrive#738] [Ideas]
Dear Community, I've been supporting and using the app for quite long now. It came to my mind, can it be possible to track the car using it's own gps data?

**Comment by hufman** [comment]
This seems quite possible, the CDS datapoints are exposed over an Android Data Provider. I built a [basic proof-of-concept AndrOBD plugin](https://github.com/BimmerGestalt/IDriveConnectAddons/blob/main/androbd_gestalt/src/main/java/io/bimmergestalt/idriveconnectaddons/androbd_gestalt/AndrobdPlugin.kt) which consumes car data and sends it on to AndrOBD, you could do something similar to save it to CSV or any other tracking app you have.
